tests/python/gaia-ui-tests/gaiatest/tests/functional/email/test_email_one_header_displayed.py

```diff
@@ -41,11 +41,9 @@ def test_only_one_header_displayed(self):
         self.device.hold_home_button()
         cards_view = CardsView(self.marionette)
         cards_view.wait_for_cards_view()
-        cards_view.wait_for_card_ready(self.email.name)
-        cards_view.close_app(self.email.name)
+        cards_view.cards[0].wait_for_centered()
+        cards_view.cards[0].close()
```

Comment:
Oh, I see what you mean. I added `wait_for_centered()` inside `close()` in case people forget to do that. It's an extra precaution.